mlos_bench/config/optimizers/one_shot_opt.jsonc

```diff
@@ -6,10 +6,10 @@
         "minimize": "score",
         // Run a single benchmark with this configuration.
         // Can be overwritten from the command line.
-        "include_tunables": [
+        "include_tunable_values": [
```

Comment:
In this case, these aren't tunables, but *values* to be assigned to tunables, so we rename the key slightly since the included json schemas are different and this makes it more clear that we're providing a mechanism for testing a particular (partial) config as well.